OpenEdXMobile/src/main/java/org/edx/mobile/util/images/ShareUtils.java

```diff
@@ -127,15 +129,49 @@ public boolean onMenuItemClick(MenuItem item) {
         menuHelper.show();
     }
 
+    public static void showCelebrationShareMenu(Activity activity, View anchor,
+                                                EnrolledCoursesResponse course,
```

Comment:
IMO, we should only pass the required parameters instead of the whole data object

Reply:
We can but it also increases the number of parameters, and also we already passing 5 parameters in number.